x-pack/plugins/observability_solution/slo/public/hooks/use_fetch_slo_burn_rates.ts

```diff
@@ -47,7 +47,11 @@ export function useFetchSloBurnRates({
           const response = await http.post<GetSLOBurnRatesResponse>(
             `/internal/observability/slos/${slo.id}/_burn_rates`,
             {
-              body: JSON.stringify({ windows, instanceId: slo.instanceId ?? ALL_VALUE }),
+              body: JSON.stringify({
+                windows,
+                instanceId: slo.instanceId ?? ALL_VALUE, // TODO Kevin: We should always have an instanceId since we backfill it from the find and get APIs
```

Comment:
Is this still TODO?

Reply:
Yes, I'm going to create an issue for all instanceId refactor